### PR TITLE
Msal-browser - Adds getLogger and setLogger

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR adds `getLogger` and `setLogger` functions to msal-browser and exports them.

### CODE REVIEW

File: lib/msal-browser/test/app/PublicClientApplication.spec.ts [hunk 1]
```diff
@@ -1924,4 +1924,27 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
             pca.emitEvent(EventType.LOGIN_START, InteractionType.Silent, {scopes: ["user.read"]}, null);
         });
     });
+
+    describe.only("Logger", () => {
```
Comment (pkanher617) [inline lib/msal-browser/test/app/PublicClientApplication.spec.ts:1928]: make sure to remove the .only here

Comment (jo-arroyo) [reply lib/msal-browser/test/app/PublicClientApplication.spec.ts]: Yes! Thank you!

**Review by jasonnutter — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

File: change/@azure-msal-browser-2020-10-28-17-06-48-msal-browser-set-logger.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Add getLogger and setLogger to msal-browser (#2513)",
+  "packageName": "@azure/msal-browser",
+  "email": "joarroyo@microsoft.com",
+  "dependentChangeType": "patch",
```
Comment (hectormmg) [inline change/@azure-msal-browser-2020-10-28-17-06-48-msal-browser-set-logger.json:6]:
```suggestion
  "dependentChangeType": "none",
```

**Review by hectormmg — APPROVED** [review]
LGTM, just have to change the dependentChangeType.

**Review by pkanher617 — APPROVED** [review]

**Comment by sameerag** [comment]
Can this be added for `msal-node` too?

**Comment by jo-arroyo** [comment]
> Can this be added for `msal-node` too?

Sure! I can add it in another PR.

**Pull request merged**